src/curated-corpus/components/ImageUpload/ImageUpload.tsx

```diff
@@ -161,12 +170,7 @@ export const ImageUpload: React.FC<ImageUploadProps> = (props): JSX.Element => {
           <Grid item xs={12}>
             <Dropzone onDrop={onDrop} maxFiles={1} accept="image/*">
               {({ getRootProps, getInputProps }) => (
-                <Box
-                  {...getRootProps({
-                    className: classes.dropzone,
-                  })}
-                  pt={2}
-                >
+                <StyledDropzoneBox {...getRootProps()} pt={2}>
```

Comment:
We already had the new styled dropzone component in `ImageUpload` over in the Collections part of the app, so I reused it here.